test/tts.hpp

```diff
@@ -107,7 +107,10 @@ namespace tts::detail
     std::string     name;
     tts::callable   behaviour;
   };
-  inline std::vector<test> suite = {};
+#ifdef __clang__ // static-initialization-order-fiasco wrkrnd (actually required only for clang-cl)
+  __attribute__(( init_priority( 101 ) ))
```

Comment:
Care to push that as an issue on TTS too ?

Reply:
can try sometime later - i'm finally returning to this area of work and have a lot catching up to do...
thanks for your work in the meantime - but please take a look at my latest changes in this pr - the problems i ran into and had to quick fix - so we can see how to fix those properly...

Reply:
have you considered [this approach](https://ctrpeach.io/posts/cpp20-string-literal-template-parameters) for rbr str to eliminate RBR_MAX_LITERAL_SIZE?

Reply:
Well, I was doing that before and went for the macro based thing to save on symbol size and debug output.

Reply:
and for the issue about init-order, I fixed TTS already so I guess it'll be uneeded as soon as I use the new TTS here.

Reply:
ok great at least something i can skip :)